Sources/TSCBasic/Process.swift

```diff
@@ -13,7 +13,7 @@ import var Foundation.NSLocalizedDescriptionKey
 import class Foundation.NSLock
 import class Foundation.ProcessInfo
 
-#if os(Windows)
+#if os(Windows) || USE_FOUNDATION
```

Comment:
do we need `if os(Windows)`? probably can just set `USE_FOUNDATION` when on windows?

Reply:
I don't think there's an equivalent to `#define` in Swift, so that would mean everyone building on Windows would need to set it manually, right?

Reply:
I was thinking that 

```
swiftSettings: [
  .define("USE_FOUNDATION")
]
```

is applied on the relevant platforms

Reply:
Ah yah, I guess we have platform conditionals for that.

Reply:
Actually, I think it's correct the way it is. `USE_FOUNDATION` is optional for non-Windows, but required for Windows which the current `#if` condition should make clear. 

If we move this into the manifest, it suggests that it is optional to pass `USE_FOUNDATION` on Windows which it isn't.